container_images/gointegtest/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM gcr.io/compute-image-tools/daisy:latest
+FROM golang
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+
+COPY --from=0 /daisy /daisy
+# Daisy image copies this from gcr.io/distroless/base, we copy here.
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+COPY --from=1 /go /go
+COPY --from=1 /usr/local/go /usr/local/go
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+# Go test junit xml output
+RUN go get -u github.com/jstemmer/go-junit-report
+
+RUN apt-get update && apt-get -y install bash ca-certificates curl libssl-dev wget && \
+    rm -rf /var/cache/apt/archives
+
+COPY . .
+
+ENTRYPOINT ["./main.sh"]
```

Comment:
Any reason to not use an absolute path? That protects you from invocations that use a different working directory.

Reply:
yeah all of our container images use this entrypoint bc the working dir might differ depending on what layer each are starting from. so this one will always work

Reply:
the entrypoint happens at runtime though, right? But the absolute path to your main.sh file is established at build time.

If you wanted to work from any entrypoint, I'd assume that an absolute path would be the safest option.